enterprise/api/v1/urls.py

```diff
@@ -7,16 +7,15 @@
 
 from enterprise.api.v1 import views
 
-# Pylint considers module level variable as "Constant" and expects them to be upper-cased
-# that is why we have disabled 'invalid-name' check for variable definition below.
 router = DefaultRouter()  # pylint: disable=invalid-name
 router.register("site", views.SiteViewSet, 'site')
 router.register("auth-user", views.UserViewSet, 'auth-user')
-router.register(
-    "enterprise-course-enrollment",
-    views.EnterpriseCourseEnrollmentViewSet,
-    'enterprise-course-enrollment'
-)
+router.register("enterprise-catalog", views.EnterpriseCustomerCatalogViewSet, 'enterprise-catalog')
+router.register("enterprise_catalogs", views.EnterpriseCustomerCatalogApiViewSet, 'enterprise_catalogs')
```

Comment:
@itsjeyd @brittneyexline Honestly I prefer dashes myself, because, well, I love `ctrl+delete`, and multiple words concatenated with underscores get insta-deleted with that keybind, but not with dashes. But since @douglashall wrote the original spec for this and he may have more insight into why he chose it (if it's not just an arbitrary choice), I'd like to hear it before I make the change, since it affects some tests I wrote, too.